src/block/node/index.ts

```
@@ -59,6 +60,7 @@ export const convertToNodes = combineNodeParsers(
   StrongImageNodeParser,
   StrongIconNodeParser,
   StrongNodeParser,
+  YoutubeNodeParser,
```

When we remove the line, the jest error is suppressed.